test/http-client.spec.js

```diff
@@ -534,21 +534,18 @@ describe('http client', () => {
 
     });
 
-    it('should reject when aborted', (done) => {
-      // TODO: Enable and fix this test when
-      // https://github.com/jasmine/jasmine-ajax/issues/111
-      // will be fixed
+    fit('should reject when aborted', (done) => {
```

Comment:
Accidental indeed. Thanks. I can't see the formatting issues myself, but if you'd show me a perfectly formatted spec I'll gladly put this and all other tests up to that standard.